CollectionViewToolsTests/CollectionViewManager/CollectionEdditingTests.swift

```diff
@@ -0,0 +1,315 @@
+//
+//  CollectionEdditingTests.swift
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import CollectionViewTools
+
+final class CollectionEdditingTests: CollectionViewTests {
+
+    func testCellRemoving() {
+        // Given
+        let strings = ["1", "2", "3"]
+        let title = "Title"
+        let reusableViewItem = TestReusableViewItem(title: title)
+        let kind = reusableViewItem.type.kind
+        // When
+        let cellItems = strings.map { string in
+            TestCollectionViewCellItem(text: string)
+        }
+        viewController.manager.sectionItems = [GeneralCollectionViewSectionItem(cellItems: cellItems,
+                                                                                reusableViewItems: [reusableViewItem])]
+        wait(for: [expectation], timeout: 10)
```

Comment:
I guess we should reduce timeouts

Reply:
If the code works it should not matter

Reply:
I agree with @artemnovichkov, every redundant second of timeout increases total time of tests to be passed